nanome_docking/autodock4/calculations.py

```diff
@@ -153,9 +154,15 @@ def _start_vina(self, receptor_file_pdbqt, ligand_file_pdbqt, num_modes=5, exhau
             args.extend(['--seed', seed])
 
         nanome.util.Logs.message("Autodock4 calculation started.")
-        process = subprocess.Popen(args, cwd=self.temp_dir, stdout=subprocess.PIPE)
+        # process = subprocess.Popen(args, cwd=self.temp_dir, stdout=subprocess.PIPE)
         self.handle_loading_bar(process, 1)
-        process.wait()
+        self.loading_bar_counter = 0
+        p = Process(vina_binary, args, buffer_lines=False)
+        p.on_error = Logs.error
+        p.output_text = True
```

Comment:
```suggestion
        p = Process(vina_binary, args, output_text=True, buffer_lines=False)
        p.on_error = Logs.error
```
Can put output_text as kwarg for Process